### ISSUE TITLE
[i18n] Method to disable fixing templates automatically

### ISSUE THREAD

**Issue by abierbaum** [angular-eslint/angular-eslint#773]
**Description and reproduction of the issue**

I have a project I am porting from tslint.  I have been using the old i18n rule.  I am using vscode to edit the code in the project and I have it configured with eslint plugin to automatically fix on save.  All of this is working correctly.

@angular-eslint is correctly flagging missing i18n tags.  But I would like a way to tell it not to try to fix them.  Our convention is to flag the issues but then make sure that the development team reviewed the i18n and creates the required tags following our identifer conventions, etc.

Is there any way to tell the i18n rule not to try to fix things?  (note: I searched around for eslint in general and I can't find a way to tell a specific rule not to attempt a fix when eslint is asked to fix.  If anyone knows a way to do that, it would be a very reasonable way to solve this)

**Versions**

| package                                   | version |
| ----------------------------------        | ------- |
| `@angular-eslint/eslint-plugin-template`  | `12.6.1` |
| `@angular-eslint/template-parser`         | `12.6.1 |
| `@typescript-eslint/parser`               | `4.28.2` |
| `ESLint`                                  | `7.32.0` |
| `node`                                    | `14.18.1` |

```sh
Angular CLI: 12.2.10
Node: 14.18.1
Package Manager: yarn 1.22.15
OS: linux x64

Angular: 12.2.10
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, localize, platform-browser
... platform-browser-dynamic, pwa, router, service-worker

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1202.10
@angular-devkit/build-angular   12.2.10
@angular-devkit/core            12.2.10
@angular-devkit/schematics      12.2.10
@angular/cdk                    12.2.11
@angular/material               12.2.11
@schematics/angular             12.2.10
rxjs                            6.6.7
typescript                      4.3.5
```

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.
